src/test/ts/browser/EditorBehaviorTest.ts

```diff
@@ -46,8 +54,8 @@ UnitTest.asynctest('EditorBehaviorTest', (success, failure) => {
           onEditorChange: eventStore.createHandler('onEditorChange')
         }),
 
-        cEditor(ApiChains.cSetContent('<p>Initial Content</p>')),
-        cEditor(ApiChains.cSetContent('<p>Initial Content</p>')), // Repeat
+        cEditor(ApiChains.cSetContent('<p>Initial Content</p>') as unknown as Chain<TinyMCEEditor, TinyMCEEditor>),
```

Comment:
Why do we need the cast?

If it's unavoidable, can you factor the casts out to a function in this module?

e.g.
```
const setContent = (s: string) => ApiChains.cSetContent(s) as unknown as Chain<TinyMCEEditor, TinyMCEEditor>`
```

Reply:
The cast exists because we use fake types for TinyMCE in mcagar. It's hard to get rid of those fake types because mcagar can't depend on the TinyMCE module as it creates a circular dependency due to TinyMCE useing mcagar for its tests. 

I will add your suggested change and a comment noting it should be removed when mcagar is updated with the true types.